src/elm/Page/Community/Editor.elm

```diff
@@ -140,6 +142,7 @@ editForm community =
     , logoList = logoList
     , inviterReward = community.inviterReward
     , invitedReward = community.invitedReward
+    , minBalance = community.minBalance |> Maybe.withDefault -100
```

Comment:
Here we have `-100` as the default value, but for the `newForm` we have `-10`.

Maybe we can do this?
```elm
, minBalance = community.minBalance |> Maybe.withDefault newForm.minBalance
```

Reply:
Done, this is way better